core/src/plugins/filed/python/vmware/bareos-fd-vmware.py

```diff
@@ -177,7 +177,7 @@ def parse_config_file(self):
         self.config = configparser.ConfigParser()
 
         try:
-            self.config.readfp(open(self.options["config_file"]))
+            self.config.read_file(open(self.options["config_file"]))
```

Comment:
this looks like `read_file()` would consumes the file object, which it doesn't.
Maybe a this is a bit more readable?
```suggestion
            with open(self.options["config_file"]) as fp:
                self.config.read_file(fp)
```

Reply:
Ok, using `with open(...)` is obviously better readable, while technically `read_file(open(...))` is ok too, it doesn't leave the file open, also appears like this in an example in the configparser docs.

Reply:
You're right, it isn't wrong. However, you have to know that a) the file is closed when the file-handle is deleted and b) ConfigParser.read_file() doesn't store the file-handle somewhere